cli/command_repository_status.go

```diff
@@ -34,6 +34,7 @@ type RepositoryStatus struct {
 
 	ClientOptions repo.ClientOptions        `json:"clientOptions"`
 	Storage       blob.ConnectionInfo       `json:"storage"`
+	Capacity      blob.Capacity             `json:"volume"`
```

Comment:
Should this be a pointer and have `omitempty` so it is not included when the capacity is unbounded? Or how should unlimited capacity be conveyed?

Reply:
Done